content/docs/run-api-server/configuring.mdx

```diff
@@ -5,35 +5,111 @@ order: 30
 
-| flag | envvar | example |
-| --- | --- | --- |
-| `--history-archive-urls` | `HISTORY_ARCHIVE_URLS` | http://history.stellar.org/prd/core-testnet/core_testnet_001/ |
-| `--ingest` | `INGEST` | true |
+Specifying command line flags every time you invoke Horizon can be cumbersome, so we recommend using environment variables. There are many tools you can use to manage them: we recommend either [direnv](http://direnv.net/) or [dotenv](https://github.com/bkeepers/dotenv). However, **if you installed via [the package manager](./installing.mdx#package-manager),** the provided `stellar-horizon-cmd` wrapper will actually import a configuration from `/etc/default/stellar-horizon` and set up the environment accordingly.
 
-Specifying command line flags every time you invoke Horizon can be cumbersome, and so we recommend using environment variables. There are many tools you can use to manage environment variables: we recommend either [direnv](http://direnv.net/) or [dotenv](https://github.com/bkeepers/dotenv). 
 
 ## Preparing the Database
 
 Before running the Horizon server, you must first prepare the Horizon database. This database will be used for all of the information produced by Horizon, notably historical information about successful transactions that have occurred on the stellar network.
 
-To prepare a database for Horizon's use, you must first ensure the database is blank. It's easiest to simply create a new database on your postgres server specifically for Horizon's use. Next you must install the schema by running `horizon db init`. Remember to use the appropriate command line flags or environment variables to configure Horizon as explained in [Configuring ](./configuring.mdx). This command will log any errors that occur.
+To prepare a database for Horizon's use, you must first ensure the database is blank. It's easiest to simply create a new database on your PostgreSQL server specifically for Horizon's use. Next, you must install the schema by running `horizon db init`. Remember to use the appropriate command line flags or environment variables to configure Horizon as explained [above](#parameters). This command will log any errors that occur.
 
 ### Postgres Configuration
+It is recommended to set `random_page_cost=1` in Postgres configuration if you are using SSD storage. With this setting, Query Planner will make a better use of indices, expecially for `JOIN` queries. We've noticed a huge speed improvement for some queries with this setting.
+
+
+## Configuring Captive Core
+
+While a full Stellar Core node requires a complex configuration with [lots of possible fields](https://github.com/stellar/stellar-core/blob/master/docs/stellar-core_example.cfg), the Captive Core configuration stub can be kept extremely barebones. The rest of the configuration will be generated automagically at runtime. Here's is a minimal working example, operating under the assumption that you want to connect to the testnet and trust SDF's validators exclusively:
```

Comment:
What would you do to configure Captive Core for the public network?  Very few people run test network Horizon instances, and I've already noticed people asking about how to adjust this config suggestion.

Reply:
I've elaborated on this slightly in a paragraph below the sample `toml` file (including a pubnet-appropriate sample here would be far too verbose imo) in e7d8d0d9399533c10a83f087d9057eb9f1aaa7c0; is that more in-line with what you were thinking, @rice2000?

Reply:
Perfect.